packages/core/src/types.ts

```diff
@@ -51,15 +51,15 @@ export type LogType =
   'recordClick.clickOccurred' |
   'recordClick.success' |
   'recordClick.callbackResponseError' |
-  'recordClick.callbackNotFoundError' |
+  'recordClick.invalidCallbackError' |
```

Comment:
Making this a bit more general now, and should now only be fired if we get to the time of recording the click or impression and find that the URL is not valid.